conf/local.conf

```diff
@@ -1,19 +1,58 @@
 # Copyright (C) 2021 AVNET Embedded, MSC Technologies GmbH
 
-MACHINE = "sm2s-imx8mm"
+MACHINE = "sm2s-am6254"
 ACCEPT_FSL_EULA = "1"
-PACKAGE_CLASSES = "package_rpm"
-EXTRA_IMAGE_FEATURES ?= "debug-tweaks"
+
+CONF_VERSION = "1"
+
+# default build to ipk. rpm create issue.
+PACKAGE_CLASSES = "package_ipk"
+
+SDKMACHINE ?= "x86_64"
+
+# We default to enabling the debugging tweaks.
+EXTRA_IMAGE_FEATURES = "debug-tweaks"
+
 USER_CLASSES ?= "buildstats image-mklibs image-prelink"
+
+# By default disable interactive patch resolution (tasks will just fail instead):
 PATCHRESOLVE = "noop"
-PACKAGECONFIG_append_pn-qemu-native = " sdl"
-PACKAGECONFIG_append_pn-nativesdk-qemu = " sdl"
-CONF_VERSION = "1"
+
+# Qemu 0.12.x is giving too much problems recently (2010.05), so disable it for users
+ENABLE_BINARY_LOCALE_GENERATION = "0"
+
+DISTRO   = "arago"
+
+# Set terminal types by default it expects gnome-terminal
+# but we chose xterm
+TERMCMD = "${XTERM_TERMCMD}"
+TERMCMDRUN = "${XTERM_TERMCMDRUN}"
+
+# Don't generate the mirror tarball for SCM repos, the snapshot is enough
+BB_GENERATE_MIRROR_TARBALLS = "0"
```

Comment:
Will not work on our buildserver, we upload the tarballs to our mirror server.

Reply:
Ok. will remove this and verified.
I have kept this as per recommended TI local.conf.